sys-apps/sg3_utils/files/sg3_utils-1.47-musl.patch

```diff
@@ -0,0 +1,11 @@
+--- a/src/sg_dd.c	2022-10-28 22:28:27.238778987 +0100
```

Comment:
1. Please run iwdevtools' scrub-patch on this
2. Include a link to patch source (ideally including metadata, get the commit url from github then append .patch to the URL, and use the contents)
3. Link to any relevant Gentoo bugs

Reply:
All areas updated as requested.

Reply:
Sorry, I mean in the patch itself at the top.